deps/wazuh_testing/wazuh_testing/tools/configuration.py

```diff
@@ -175,16 +175,16 @@ def create_elements(section: ET.Element, elements: List):
             for tag_name, properties in element.items():
                 tag = ET.SubElement(section, tag_name)
                 new_elements = properties.get('elements')
+                attributes = properties.get('attributes')
+                if attributes:
+                    for attribute in attributes:
+                        if attribute is not None and isinstance(attribute, dict):  # noqa: E501
```

Comment:
This is a redundant check. If the `attribute` is a `dict`, it will never be None.
```suggestion
                        if isinstance(attribute, dict):  # noqa: E501
```